.editorconfig

```diff
@@ -5,7 +5,6 @@ root = true
 # All files
 [*]
 indent_style = space
-end_of_line = lf
```

Comment:
Why remove this? LF is compatible with Linux, Mac and Windows (normally), whereas CRLF is Windows-only.

Git should also have files checked in as LF.

https://www.hanselman.com/blog/carriage-returns-and-line-feeds-will-ultimately-bite-you-some-git-tips

However not sure what this would entail in our particular case...

Reply:
Because I thought that all the OEL was in CRLF and not in LF (see the last false report)
And as you said to me last time, it is not the time to add too many noises on Git.

But, in fact, I a pretty sure that it is because the workflow clones the git repository in CRLF style, surely caused by the Win image.